databases-analytics/databases/postgresql-tf/my-provider.tf

```diff
@@ -0,0 +1,11 @@
+terraform {
+  required_providers {
+    ovh = {
+      source  = "ovh/ovh"
+    }
+  }
+}
+
+provider "ovh" {
+  endpoint           = "ovh-eu"
```

Comment:
instead of hardcode "ovh-eu", this public cloud example can work also in US and Canada :)

So my advice is to fill the env variable: `export OVH_ENDPOINT="ovh-eu"`

possible values: `ovh-eu`, `ovh-ca`, `ovh-us`, `soyoustart-eu`, `soyoustart-ca`, `kimsufi-ca`, `kimsufi-eu`, `runabove-ca`

Reply:
yes, and we must do the same in the TF file for MKS 😉: https://github.com/ovh/public-cloud-examples/blob/f3250c521201b241cea16905de701116af995877/containers-orchestration/managed-kubernetes/create-cluster-with-tf/my-provider.tf#L10

Reply:
done with https://github.com/ovh/public-cloud-examples/pull/38/commits/0869062790fc17697604e8f4d958738049744902

Reply:
thx and yes you right for mks :)